certificates/key.pem

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PRIVATE KEY-----
+MIIJQgIBADANBgkqhkiG9w0BAQEFAASCCSwwggkoAgEAAoICAQC6Z1oPVN14FdED
+zTV628aHR/+ZQ62KmzT5+/HcwD7vekq7iUuexPkPE8n2dy07rg7sCe/FrsnJ7uoL
+Z+p4ZFsHcrHPnaHZHuvEX1tUb515fO4R0SCKjHPnSOM8fYC1VPPmHq5YMJzSNXEj
+0N0e1hJRLgIjxCnrxGtRLxW1f6/DCyGYounnQvvmvnfWOwIWDQAm4YY6JxtI5W10
+O9jmys8XVIf37Y/wElFLv+Y0QI1iy01rcN0YcgIx2QnoDCCBZIzh+SEHMpD4xRrt
+9vLcNbKEoBaiP3QMeWKgJlUDSFvCdFnpCCXXshL9CDqARwOhdqy1+Z4tN3ihYE2o
+ErhtE4/nkfK37RobHJ3eIg8cOrXPLi+pk4GYXSLdme+WVX5+EHoTj4nwZaakWQp4
+MZKQZbk7m7FAgPqIyFQlesJi7DZWGpgT1adRLvOIQtUobOj2ysThk93ZKcDPLLPW
+xitCMyr2AZJZb0ZsoJFC2KCm1wGJRaeTxYON2CkSHW5kweDKZCMy9UF9iqnMehKj
+vuiHCPRUj9yS2vAgTNcOAvJ/QFW6VZMRaFFElrs14S+ut2sb6acQ4vZukR/p9io5
+b9T8bTyHHSzWOSEKQ3L0w23ibILnb048LQrOVVnFLoJk2c6ni6yde6KzmhmOpY3f
+SaqUmzvfg9bHlpKgHpNU6FmNwJK8dwIDAQABAoICACSh7g2ISJPyeaatG/7+lQf+
+CsnEhqTwNpuQO2p+L4AhwPEVHNLhc+7WHUE+D5okbVNO6puna4mvVbiTHf21/zu5
```

Comment:
senstiive data, please dont include these files in the commit, also you dont need these files to run the app in ssl mode

Reply:
I'm not 100% sure if all you get is a warning screen, if so you can just continue to your application